### PR TITLE
added new features tickets 2260, 4124, 2258

### PR DESCRIPTION
**Pull request by ozbenoved**
+ removed account setting from gradle + deleted unused xmls

### CODE REVIEW

File: ConversationDemo/app/src/main/java/com/conversation/demo/DemoRemoteViewHolder.java [hunk 1]
```diff
@@ -8,35 +8,56 @@
...
         }
 
-        RemoteOptionsChatElement remoteChatElement = (RemoteOptionsChatElement) element;
-        avatarImageView.setImageResource(remoteChatElement.getAgentType().equals(AgentType.Live) ?
+        ContentChatElement remoteChatElement = (ContentChatElement) element;
+
+        DynamicBubbleBind.BubbleData bubbleData = dynamicBubbleBind.onBind(remoteChatElement, position, totalCount);
+
+        if(bubbleData.displayAvatar()) {
+            avatarImageView.setImageResource(remoteChatElement.getAgentType().equals(AgentType.Live) ?
```
Comment (tehilaR2) [inline ConversationDemo/app/src/main/java/com/conversation/demo/DemoRemoteViewHolder.java:49]: look at JioRemoteViewHolder code regarding isLive check

File: ConversationDemo/app/src/main/java/com/conversation/demo/MainActivity.java [hunk 2]
```diff
@@ -72,29 +74,35 @@
...
 
     private static final String TAG = "MainActivity";
 
-
     /* --- Please fill with valid account values --- */
-    private static final String ACCOUNT_NAME = BuildConfig.ACCOUNT_NAME;
-    private static final String API_KEY = BuildConfig.API_KEY;
-    private static final String KNOWLEDGE_BASE = BuildConfig.KNOWLEDGE_BASE;
-    private static final String SERVER = BuildConfig.SERVER;
+    private static final String ACCOUNT_NAME = "";
```
Comment (tehilaR2) [inline ConversationDemo/app/src/main/java/com/conversation/demo/MainActivity.java:102]: why didn't you delete those constants

File: ConversationDemo/app/src/main/java/com/conversation/demo/DemoRemoteViewHolder.java (hunk 1 above)
Comment (ozbenoved) [reply ConversationDemo/app/src/main/java/com/conversation/demo/DemoRemoteViewHolder.java:49]: :)

File: ConversationDemo/app/src/main/java/com/conversation/demo/MainActivity.java (hunk 2 above)
Comment (ozbenoved) [reply ConversationDemo/app/src/main/java/com/conversation/demo/MainActivity.java:102]: :)

**Pull request merged**